topics/single-cell/tutorials/scrna-ncbi-anndata/tutorial.md

```diff
@@ -0,0 +1,477 @@
+---
+layout: tutorial_hands_on
+
+title: Converting NCBI Data to the AnnData Format
+subtopic: datamanipulation
+priority: 3
+zenodo_link: 'https://zenodo.org/record/10101768'
+
+questions:
+- How do I understand NCBI data?
+- How can I convert raw gene data to the AnnData format?
+- How do I manually and automatically add metadata to my AnnData object?
+objectives:
+- Analyse some gene data from NCBI.
+- Convert raw gene expression data into the AnnData format.
+- Add metadata to the AnnData object to prepare it for analysis.
+time_estimation: 1H
+key_points:
+- Single cell data often requires additional processing and conversion before it is able to be analysed.
+- Metadata and annotations can be added in both manually and automatically with various tools and resources.
+
+tags:
+- single-cell
+- data management
+
+contributions:
+  authorship:
+    - hexhowells
+
+follow_up_training:
+  -
+    type: "internal"
+    topic_name: single-cell
+
+---
+
+# Introduction
+
```

Comment:
```suggestion
```

we're removing the `# Introduction` type heading so that we can more easily automatically extract the first paragraph as an 'abstract' for the tutorial. This is a great introduction you have to the tutorial!